src/CommunityToolkit/Directory.build.props

```diff
@@ -52,4 +52,13 @@
         <Compile Include="**\*.macos.cs" />
         <Compile Include="**\*.macos.*.cs" />
     </ItemGroup>
+
+    <ItemGroup Condition=" !$(TargetFramework.Contains('-')) ">
+      <Compile Include="**\*.netstandard.cs" />
+      <Compile Include="**\*.netstandard.*.cs" />
+    </ItemGroup>
+    <PropertyGroup Condition=" !$(TargetFramework.Contains('-')) ">
+      <DefineConstants>$(DefineConstants);NETSTANDARD</DefineConstants>
+    </PropertyGroup>
```

Comment:
```xml
 <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' OR $(TargetFramework.StartsWith('netstandard'))">
    <Compile Include="**\*.netstandard.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
    <Compile Include="**\*.netstandard.*.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
  </ItemGroup>
  ```

From MAUI

Reply:
Just including files won't be enough in our case. We also have a `shared` files with `#if NETSTANDARD` check (that doesn't have an equivalent in .Net5-6), so preprocessor symbol also needs to be defined.

Regarding the condition, `!$(TargetFramework.Contains('-'))` looks cleaner in my opinion. Since we don't target any framework sdk, any TFM that doesn't have a `-` is platform-agnostic.